pkg/diff/diff.go

```diff
@@ -132,65 +79,59 @@ func WalkChan(dir string, ignores []string, latestModTime int64) <-chan *Entry {
 				return pushErr(fmt.Errorf("stat file: %w", err))
 			}
 
-			if entry.IsDir() {
-				maybeEmptyDir = relativePath
-				emptyDirMode = info.Mode()
-				return nil
+			// Fetch the inode if we can, otherwise fallback to setting it to 0
+			inode := uint64(0)
+			sysStat, ok := info.Sys().(*syscall.Stat_t)
```

Comment:
Is this gonna issue a second syscall for each file?

Reply:
As I understand it, this data comes from the same syscall but is left in this untyped object so you can cast it depending on the host OS.

All of this info comes from the same `lstat` call under the hood.

Reply:
Okey dokey!